package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "todolist",
+  "version": "1.0.0",
+  "description": "This project is an online library to add/delete and store books locally using javascript ES6 objects and arrow notation.",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --open",
```

Comment:
- [ ] Good job so far 👏 However, I can't be able to run your project because I am getting the error when I try to run `npm run start` 

I would suggest that you change it to the following 👇 
![image](https://user-images.githubusercontent.com/90222110/236564698-98cb0504-2bbb-45a5-a0c0-aa8046e0d3d7.png)

By doing this, you will be able to install and run your project successful and I guess everyone else will be able to run it 😉